resources/aws/kms.go

```diff
@@ -21,14 +27,31 @@ func (kk *KMSKey) CreateOrFail() error {
 	if err != nil {
 		return microerror.MaskAny(err)
 	}
+
+	if _, err := kk.Clients.KMS.CreateAlias(&kms.CreateAliasInput{
+		// Alias names need to start from "alias/" prefix.
+		AliasName:   aws.String(kk.fullAlias()),
+		TargetKeyId: key.KeyMetadata.Arn,
+	}); err != nil {
+		return microerror.MaskAny(err)
+	}
+
 	kk.arn = *key.KeyMetadata.Arn
+
 	return nil
 }
 
-func (kms *KMSKey) Delete() error {
-	return microerror.MaskAny(notImplementedMethodError)
+func (kk *KMSKey) Delete() error {
+	// AWS API doesn't allow to delete the KMS key itself, but we remove the alias and leave the key nameless.
```

Comment:
We can't delete the key because of the KMS API restriction. But I think there is still a cleanup problem because there is a default limit of 1000 keys per region. 

http://docs.aws.amazon.com/kms/latest/developerguide/limits.html

Can you raise an issue for the cleanup task and add to the comment?